manticore/manticore.py

```diff
@@ -290,40 +266,23 @@ def dumpafter(self, dump_after):
     def maxstorage(self):
         return self._maxstorage
 
-    @maxstorage.setter
-    def maxstorage(self, max_storage):
-        self._maxstorage = max_storage
-
     @property
     def verbosity(self):
-        '''
-        Convenience interface for setting logging verbosity to one of several predefined
-        logging presets. Valid values: 0-5
-        '''
-        return self._verbosity
+        """Interface for getting Manticore logging verbosity.
```

Comment:
super nit here: for properties, our auto generated docs only show the docstring for the getter, not the setter. so we need the getter docstring to also express that you can set it here. 
 (http://manticore.readthedocs.io/en/latest/api.html#manticore.Manticore.verbosity) Can we copy verbatim what we used to have, since I don't see a reason to change the language?